prelude/src/stdlib/ffi.sk

```diff
@@ -0,0 +1,7 @@
+module FFI;
+
+// TODO: This should probably return a Result<> in case invalid utf8 is provided.
+@cpp_extern
```

Comment:
Why not `@cpp_extern("sk_string_create")` here and then no need to define `SKIP_FFI_utf8_bytes_to_string` in string.c?